projects/kserve/testing/test_e2e.py

```diff
@@ -324,10 +324,15 @@ def deploy_consolidated_model(consolidated_model, namespace=None, mute_logs=None
         logging.warning("No secret env key defined for this model")
 
 
-    if (extra_env := consolidated_model["serving_runtime"].get("extra_env")):
+    if (extra_env := consolidated_model["serving_runtime"]["transformer"].get("extra_env")):
```

Comment:
A nit from your past self ;) 

```suggestion
    if (extra_env := consolidated_model["serving_runtime"].get("transformer", {}).get("extra_env")):
```
If the transformer or kserve dicts are missing we will get a keyerror.

Reply:
😀